### PR TITLE
fix copy/paste check across multiple tabs

### PR DESCRIPTION
**Pull request by manavortex**
# copy/paste check across multiple tabs

### CODE REVIEW

**Comment by rfuzzo** [comment]
One thing I noticed is that the event is raised three times when right clicking on one chunk "Copy from array/buffer"

image

**Comment by manavortex** [comment]
> One thing I noticed is that the event is raised three times when right clicking on one chunk "Copy from array/buffer"
> 
> image

Did you copy three chunks?

**Comment by rfuzzo** [comment]
No, it's just one chunk. It happens because the eventhandler is leaking. We never remove the event binding, but each newly opened file adds a new handler.

File: WolvenKit/Views/Tools/RedTreeView.xaml.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public RedTreeView()
             InitializeComponent();
 
             DataContextChanged += OnDataContextChanged;
-            RedDocumentTabViewModel.OnCopiedChunkChanged += AfterCopied_RefreshCommandStatus;
```
[the remark this answers is not in the document]
Comment (manavortex) [reply WolvenKit/Views/Tools/RedTreeView.xaml.cs:51]: When do I even unsubscribe? Not seeing a teardown

[the remark this answers is not in the document]
Comment (rfuzzo) [reply WolvenKit/Views/Tools/RedTreeView.xaml.cs:51]: Yeah it's not straightforward. The usercontrol get's disposed at random times when the Garbage Collector thinks it's opportune :D One way could be to unsubscribe in the `Unloaded` event of the RedTreeView, but that's also problematic as the user control may get unloaded and reactivated 🤔

[the remark this answers is not in the document]
Comment (rfuzzo) [reply WolvenKit/Views/Tools/RedTreeView.xaml.cs:51]: see https://github.com/WolvenKit/WolvenKit/pull/2082

**Comment by rfuzzo** [comment]
This needs to wait for https://github.com/WolvenKit/WolvenKit/pull/2082 to be resolved

**Pull request merged**